### DISCUSSION TITLE
send email to manager when changing passwords?

### DISCUSSION THREAD

**Discussion by s1vestre** [nilsteampassnet/TeamPass#3792] [Q&A]
send email to manager when changing passwords? it's possible?

**Comment by nilsteampassnet** [comment]
@s1vestre 
You mean when any item password is changed?

**Reply by Arash9703** [reply]
@nilsteampassnet , Do you have any suggestions for customizing email templates ( content of password reset emails) Even after altering the Teampass language settings within the application, the content of the reset emails remains unchanged 
[https://github.com/nilsteampassnet/TeamPass/discussions/3958#discussion-5895598](url)
Thanks
